### DISCUSSION TITLE
Suggestions for update frequency

### DISCUSSION THREAD

**Discussion by andersrothman-coop** [alandtse/tesla#96] [Ideas]
It would be nice to have two polling intervals. One when connected and one when disconnected.

**Comment by alandtse** [comment]
There already is. When disconnected we check every minute to see if the car wakes up. This is done to Tesla directly. In fact, that check with Tesla is continuous. The scan interval options only impact connections to the car itself.

**Reply by andersrothman-coop** [reply]
thanks
